src/frontend/src/flows/faq/index.ts

```diff
@@ -1,32 +1,50 @@
 import { html, render } from "lit-html";
+import "../../styles/main.css";
```

Comment:
```suggestion
```

Since the faq page is rendered from `frontend/src/index.ts` the css should already be around.